python/google/protobuf/pyext/extension_dict.h

```diff
@@ -47,7 +47,8 @@ namespace protobuf {
 class Message;
 class FieldDescriptor;
 
-using internal::shared_ptr;
+using std::shared_ptr;
```

Comment:
Why did you change this? We are trying to stay compatible with C++98, which does not have shared_ptr in std.